lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize
+    @dishes = {
+      "Margherita" => 5.00,
+      "Pepperoni" => 8.00
+    }
```

Comment:
Would a single hash be the best way to store this data? For allowing this class to be expanded later an array of hashes could be better.